propose_sns.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+#
+# This takes an optional path to a configuration file; defaults to
+# sns_init.yaml, in the same directory as this script. This argument gets passed
+# to `sns propose`. Such a file can be constructed by following the directions
+# at the top of example_sns_init.yaml.
+#
+# This is mostly copied from deploy_sns.sh.
+#
+# Besides replacing `sns deploy` with `sns propose`, the biggest differences
+# here are
+#
+#     1. No more `ic-admin ... propose-to-update-sns-deploy-whitelist`.
+#
+#     2. This does not take a CONFIG argument.
+
+set -euo pipefail
+
+cd -- "$(dirname -- "${BASH_SOURCE[0]}")"
+
+export SNS_CONFIGURATION_FILE_PATH="${1:-sns_init.yaml}"
+
+. ./constants.sh normal
+
+export CURRENT_DX_IDENT="$(dfx identity whoami)"
+
+dfx identity use "${DX_IDENT}"
+
+. ./setup_wallet.sh "${DX_IDENT}"
+
+# Conjur some cycles.
+dfx ledger \
+    --network "${NETWORK}" \
+    fabricate-cycles \
```

Comment:
these cycles (and the above wallet) are presumably not needed anymore (as the wallet is not used to make a call to the SNS-W canister anymore)

Reply:
the wallet is actually needed to transfer cycles to SNS-W using deposit-cycles dfx command